Makefile

```diff
@@ -90,7 +90,6 @@ check-examples: valadoc-example-tester
 #
 
 build-data:
-	npm install
```

Comment:
Did you mean to commit this?

Reply:
Then we should update the README or add it to another step in the `Makefile`. If someone clones it and runes `make serve`, it will fail.

Reply:
:+1: